README.md

```diff
@@ -62,17 +62,17 @@ Other paths to files should be set according to your configuration, using a file
 
 In one terminal window, open the snap daemon (in this case with logging set to 1,  trust disabled and global configuration saved in config.json ):
 ```
-$ $SNAP_PATH/bin/snapd -l 1 -t 0 --config config.json
+$ $SNAP_PATH/bin/snapteld -l 1 -t 0 --config config.json
```

Comment:
Now path should be without ''bin", so it should be
```
 $ $SNAP_PATH/snapteld -l 1 -t 0 --config config.json
```
In my environment I have binary files in this localization
```
 /home/kzab/work/src/github.com/intelsdi-x/snap/build/linux/x86_64
```